workflow/controller/operator_test.go

```diff
@@ -4101,7 +4101,7 @@ status:
 		defer cancel()
 		woc := newWorkflowOperationCtx(wf, controller)
 		woc.operate()
-		assert.Equal(t, wfv1.NodePending, woc.wf.Status.Phase)
+		assert.Equal(t, wfv1.NodeRunning, woc.wf.Status.Phase)
```

Comment:
Resubmit will memoization wf phase should be running after `operate` function. I will create separate PR to fix the phase in memorization

Reply:
ok

Reply:
if there is another bug fix - can we combine into a single valid PR?

Reply:
sure I will fix the memoization bug also in this PR.

Reply:
As per @jessesuen discussed, just need to fix the phase in resubmit memorized scenario.